snapcraft.yaml

```diff
@@ -60,3 +65,35 @@ parts:
     build-packages:
       - python3-pip
       - python3-setuptools
+      - libmagic-dev
+      - pkg-config
+      - libzip-dev
+      - liblz4-dev
+      - libssl-dev
+      - libbsd-dev
+      - libcapstone-dev
+      - cmake
```

Comment:
why cmake?

Reply:
`cmake` was a remnant from an earlier version of the snapcraft.yaml configuration. 
I removed it. 

The `build-packages` are packages that are installed to aid in the compilation of the software.